tests/core/util/test_streamable.py

```diff
@@ -84,74 +92,83 @@ class TestClassPlain(Streamable):
             a: PlainClass
 
 
-def test_basic_list():
+def test_basic_list() -> None:
     a = [1, 2, 3]
     assert is_type_List(type(a))
     assert is_type_List(List)
     assert is_type_List(List[int])
     assert is_type_List(List[uint8])
     assert is_type_List(list)
-    assert not is_type_List(Tuple)
+    assert not is_type_List(type(Tuple))
     assert not is_type_List(tuple)
     assert not is_type_List(dict)
 
 
-def test_not_lists():
+def test_not_lists() -> None:
     assert not is_type_List(Dict)
 
 
-def test_basic_optional():
-    assert is_type_SpecificOptional(Optional[int])
-    assert is_type_SpecificOptional(Optional[Optional[int]])
+def test_basic_optional() -> None:
+    # Below mypy failure: Argument 1 to "is_type_SpecificOptional" has incompatible type "object"; expected "Type[Any]"
+    assert is_type_SpecificOptional(Optional[int])  # type: ignore[arg-type]
```

Comment:
I'm not quite sure but it may be that these functions should be hinted to accept `object` instead.  Hinting acceptance of 'a hint' is I think a thing that is not well defined at present.  In the end, _anything_ can be put in an annotation.

Reply:
See [1bbeacb](https://github.com/Chia-Network/chia-blockchain/pull/10539/commits/1bbeacb42c13ce52511ed7d1043b32475d409016)